templates/tv/package.json

```
@@ -3,11 +3,10 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "android": "react-native run-android",
-    "tvos": "arch -x86_64 react-native run-ios  --simulator \"Apple TV 4K (2nd generation)\" --scheme \"tvappnative-tvOS\"",
+    "androidtv": "emulator -avd \"$(emulator -list-avds | grep TV)\" & react-native run-android",
```

"androidtv" breaks yarn tv:android script in the root package.json.

```suggestion
    "android": "emulator -avd \"$(emulator -list-avds | grep TV)\" & react-native run-android",
```